src/app/services/local-storage.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { IListOfQuestions, IQuestionInfo } from '../interfaces/question.interfaces';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class localStorageService {
```

Comment:
localStorage service in this task should replace http, so it has setToLocalStorage, getFromLocalStorage services
for others you create apropriate services (i.e. QuestionsService, which has createQuestion, deleteQuestion, answer, delete answer etc.)
So in future when we have working BE it's easy to replace from mock LocalStorageService
please update it